fast64_internal/oot/oot_f3d_writer.py

```diff
@@ -11,6 +11,27 @@
 from .oot_model_classes import *
 from .oot_scene_room import *
 
+
+class OOTDLExportSettings(bpy.types.PropertyGroup):
+    name: bpy.props.StringProperty(name="DL Name", default="gBoulderFragmentsDL")
+    folder: bpy.props.StringProperty(name="DL Folder", default="gameplay_keep")
+    customPath: bpy.props.StringProperty(name="Custom DL Path", subtype="FILE_PATH")
+    isCustom: bpy.props.BoolProperty(name="Use Custom Path")
+    removeVanillaData: bpy.props.BoolProperty(name="Replace Vanilla DLs")
+    drawLayer: bpy.props.EnumProperty(name="Draw Layer", items=ootEnumDrawLayers)
+    assetIncludeDir: bpy.props.StringProperty(name="Asset Include Directory", default="assets/objects/gameplay_keep")
```

Comment:
It's unclear to me what `assetIncludeDir` is, it's the base path for `#include` directives, for custom paths exports?

if that's right,
maybe -> `customAssetIncludeDir`
and `description="Used in #include for including exported files"` (?)

Reply:
Yeah, its the base path for including image files.